dashboard/modules/state/state_head.py

```diff
@@ -192,9 +192,16 @@ def _options_from_req(self, req: aiohttp.web.Request) -> ListApiOptions:
         timeout = int(req.query.get("timeout", 30))
         filters = self._get_filters_from_req(req)
         detail = convert_string_to_type(req.query.get("detail", False), bool)
+        show_dead_jobs = convert_string_to_type(
```

Comment:
Let's just make it true for state API? I think we will sort the data to

alive actors -> dead actors 

before reporting it, so truncate should have the same effect all the time.

Reply:
in that case, should we just set it True when querying GCS? And not make it configurable?

Reply:
yes! I think there's no need to make it configurable (actually we should remove this flag. this is weird)

Reply:
Do you mean remove this flag altogether and make it default to return dead jobs?

Reply:
yes. OR we can just filter by states (like source-side filtering). I think this was the hack to reduce the returned # of actors